TCF4.py

```diff
@@ -0,0 +1,121 @@
+from Bio import SeqIO
+from Bio.SeqUtils import GC
+
+def filter_fastq(input_path: str, quality_threshold: int, output_filename="final_filtered.fastq",  gc_bounds=(40, 60), length_bounds=(50, 350)):
+    filename = input_path
+    records = SeqIO.parse(filename, "fastq")
```

Comment:
Зачем вводить переменную `filename`? Так ты просто заставляешь два имени ссылаться на один и тот же объект. Можно было оставить `input_path`:
```suggestion
    records = SeqIO.parse(input_path, "fastq")
```